devices.js

```diff
@@ -8178,6 +8178,25 @@ const devices = [
         },
     },
 
+    {
+        zigbeeModel: ['CurtainMo-EF'],
+        fingerprint: [
+            {modelID: 'CurtainMo-EF', manufacturerName: 'HEIMAN'},
```

Comment:
Why have both `fingerprint` and `zigbeeModel` here? `zigbeeModel` is enough I think

Reply:
thinks for advice